src/System/executable_query.rs

```diff
@@ -86,7 +86,7 @@ pub struct ReachabilityExecutor {
     // e_state is the end state, where we want to see whether end state is reachable from start state
     pub end_state: State,
 }
-
+#[allow(unused_variables)]
 impl ExecutableQuery for ReachabilityExecutor {
     fn execute(self: Box<Self>) -> QueryResult {
         let (sys, s_state, e_state) = (self.transition_system, self.start_state, self.end_state);
```

Comment:
You should delete line 92 instead of allowing unused variables.

Reply:
We dont want warnings here so we allow the unused variables. Another PR with the searching algo for reachability will soon be up without this.

Reply:
Then don't fix the warning in this pr

Reply:
Fixed @mads256h